data.sql

```diff
@@ -0,0 +1,14 @@
+-- day 1
+INSERT INTO pets (name, birth_date, escape_attempts, neutered, weight_kg) VALUES ('Agumon', DATE '2020-02-03', 0, true, 10.23);
+INSERT INTO pets (name, birth_date, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', DATE '2018-11-15', 2, true, 8);
+INSERT INTO pets VALUES (3, 'Pikachu', DATE '2021-01-07', 1, false, 15.04);
+INSERT INTO pets VALUES (4, 'Devimon', DATE '2017-05-12', 5, true, 11);
+
+-- day 2
+INSERT INTO pets (id, name, birth_date, escape_attempts, neutered, weight_kg) VALUES (5, 'Charmander', DATE '2020-02-08', 0, false, 11);
+INSERT INTO pets (id, name, birth_date, escape_attempts, neutered, weight_kg) VALUES (6, 'Plantmon', DATE '2021-11-15', 2, true, 5.7);
+INSERT INTO pets (id, name, birth_date, escape_attempts, neutered, weight_kg) VALUES (7, 'Squirtle', DATE '1993-04-02', 3, false, 12.13);
+INSERT INTO pets (id, name, birth_date, escape_attempts, neutered, weight_kg) VALUES (8, 'Angemon', DATE '2005-06-12', 1, true, 45);
```

Comment:
- [ ] Please kindly add the `-` to your `weight` as asked on the project requirements the values are `-11.0, -5.7, -12.13, -45.0` 🙃 🙃